docs/_templates/page.html

```diff
@@ -52,6 +52,18 @@
         alt="Funded by the European Union"
       />
     </div>
+    <div class="acknowledgement only-light">
+      <img
+        src="{{ pathto('_static/unitary_logo.svg', 1) }}"
+        alt="Supported by Unitary Fund"
+      />
+    </div>
+    <div class="acknowledgement only-dark">
+      <img
+        src="{{ pathto('_static/unitary_logo.svg', 1) }}"
+        alt="Supported by Unitary Fund"
+      />
+    </div>
```

Comment:
No need to distinguish if you are using the same logo in light and dark mode.
```suggestion
    <div class="acknowledgement">
      <img
        src="{{ pathto('_static/unitary_logo.svg', 1) }}"
        alt="Supported by the Unitary Fund"
      />
    </div>
```